### ISSUE TITLE
bug(web): Console errors on Android API 21

### ISSUE THREAD

**Issue by darcywong00** [keymanapp/keyman#12189]
While restesting #12185, I accidentally started an old Android API 21 emulator

Keyman had a blank keyboard with the following console errors (repeatable on `master` branch)

```
Uncaught SyntaxError: Unexpected token ) keymanweb-webview.js:2
Uncaught SyntaxError: Unexpected token ) keyman-sentry.js:1
Uncaught ReferenceError: KeymanSentryManager is not defined android-host.js:17
```

I haven't tracked down how long it's been like this.

**Comment by darcywong00** [comment]
17.0.328 stable loads fine

**Comment by darcywong00** [comment]
I'd rather this go to someone in KeymanWeb since they're likely to know what recent refactoring could have caused this...

**Comment by jahorton** [comment]
We dropped ES5 downcompilation; Android 5.0 devices + images will need a Chrome update in order to properly run Keyman.

**Comment by darcywong00** [comment]
Closing as won't fix:  Minimum Chrome version needs to be 53.0 for KeymanWeb in 18.0
